main/options/options.cc

```diff
@@ -276,7 +276,7 @@ buildOptions(const vector<pipeline::semantic_extension::SemanticExtensionProvide
     // Used to populate default options.
     Options empty;
 
-    cxxopts::Options options("sorbet", "Typechecker for Ruby");
+    cxxopts::Options options("srb", "Typechecker for Ruby");
```

Comment:
Could you please remove this from the PR?
for the same reason we've pushed back against it in past: it'll work incorrecly in `srb` too, as you want `srb tc`(or other wrapper commands).

The proper way it to allow to override command name via options

Reply:
sorry